packages/@aws-cdk/integ-runner/test/test-data-typescript/xxxxx.typescript-test.ts

```diff
@@ -0,0 +1,9 @@
+import { App, Stack } from '@aws-cdk/core';
```

Comment:
This one is okay to be named `integ.typescript-test.ts`

(The other ones aren't because they are empty files that only need to exist for mocking purposes and we should probably try to just get rid of them at some point)